build_variables.bzl

```diff
@@ -675,107 +675,6 @@ libtorch_cuda_core_sources = [
     "torch/csrc/jit/codegen/fuser/cuda/fused_kernel.cpp",
     "torch/csrc/profiler/stubs/cuda.cpp",
     "torch/csrc/autograd/functions/comm.cpp",
-    "torch/csrc/jit/codegen/cuda/arith.cpp",
```

Comment:
Hmm, looks like this change makes nvfuser inaccessible to bazel (and therefore internally) Is this intentional?
cc: @davidberard98

Reply:
We are aware of this and do plan on adding bazel build back in future PR, have a comment in the PR

> Since we moved nvfuser into a cmake build, we effectively disabled bazel build for nvfuser. This could impact internal workload at Meta, so we need to put support back. cc'ing @vors